app/src/features/projects/components/ProjectCoordinatorForm.test.tsx

```diff
@@ -0,0 +1,57 @@
+import renderer from 'react-test-renderer';
+import ProjectCoordinatorForm, {
+  ProjectCoordinatorInitialValues,
+  ProjectCoordinatorYupSchema
+} from 'features/projects/components/ProjectCoordinatorForm';
+import { Formik } from 'formik';
+import React from 'react';
+
+const handleSaveAndNext = jest.fn();
+
+const ProjectCoordinatorFilledValues = {
+  first_name: 'Nerea',
+  last_name: 'Oneal',
+  email_address: 'quxu@mailinator.com',
+  coordinator_agency: 'Qui anim qui laboris',
+  share_contact_details: 'true'
+};
+
+describe('Project Coordinator Form', () => {
+  it('renders correctly the empty component correctly', () => {
+    const emptyComponent = renderer
```

Comment:
discussing this with nick on other PR, but I think we should stick to using the `react-testing-library` and its utilities

Reply:
`@testing-library/react`*

Reply:
@anissa-agahchen We are going to move to using the @testing-library/react library instead of the react-testing-library.  The tests themselves pretty much dont't change, we just have to change the `render` call tot he new syntax.

Can see examples in my PR: https://github.com/bcgov/biohubbc/pull/96/files#diff-995347c828b01a24b902322d2a089545610ac099222f237f26e7e7549a500f1b

Reply:
done